jsonLoader.go

```diff
@@ -334,6 +345,7 @@ type jsonRawLoader struct {
 	source interface{}
 }
 
+// NewRawLoader creates a new JSON raw loader for the given source
 func NewRawLoader(source interface{}) *jsonRawLoader {
```

Comment:
All the other loaders had their type changed from `*jsonRawLoader` to `JSONLoader`. Apparently this one was missed, but it can be safely changed to `JSONLoader` as a return type which solves one more linting issue.

Reply:
Ah! Yes, makes sense; updating